documentation/src/main/asciidoc/stories/assembly_cli_command_reference.adoc

```diff
@@ -32,7 +32,7 @@ include::{cli_help}/help.adoc[leveloffset=+1]
 include::{cli_help}/ls.adoc[leveloffset=+1]
 include::{cli_help}/patch.adoc[leveloffset=+1]
 include::{cli_help}/put.adoc[leveloffset=+1]
-include::{cli_help}/query.adoc[leveloffset=+1]
+include::{cli_help}/search.adoc[leveloffset=+1]
```

Comment:
asciidoctor: ERROR: stories/assembly_cli_command_reference.adoc: line 35: include file not found: /home/dnaro/git/dnaro-query-review/cli/cli-client/src/main/resources/help//search.adoc

This should change back to query.adoc because references this file: https://github.com/infinispan/infinispan/blob/master/cli/cli-client/src/main/resources/help/query.adoc 

That's the manpage for the query command. We include it in the CLI guide with the command reference section.

I would agree that we should probably change "query" to "search" but that's probably out of scope at this point.

Reply:
ops